examples/chart/teleport-cluster/charts/teleport-operator/README.md

```diff
@@ -0,0 +1,28 @@
+# Teleport Operator
+
+This chart deploys the Teleport Kubernetes Operator. The operator allows to manage
+Teleport resources from inside Kubernetes.
+
+## Important notice
+
+The chart version follows the Teleport and Teleport Kube Operator version. e.g.
+chart v15.0.1 runs the operator version 15.0.1 by default. To control which
+operator version is deployed, use the `--version` Helm flag.
+
+## Deployment
+
+The chart can be deployed in two ways:
+- in standalone mode by running
+  ```shell
+  helm install teleport/teleport-operator teleport-operator --set authAddr=teleport.example.com:443 --set token=my-operator-token
+  ```
+  See [the standalone guide](https://goteleport.com/docs/management/dynamic-resources/teleport-operator-standalone/) for more details.
+- as a dependency of the `teleport-cluster` Helm chart by adding `--set operator.enabled=true`. See
+  [the operator within teleport-cluster chart guide](https://goteleport.com/docs/management/dynamic-resources/teleport-operator-helm/).
```

Comment:
I wonder if we should omit this part

Reply:
I was assuming we want to standardize in a single method for deploying the operator.
Maybe I'm wrong, sorry.

Reply:
If so, we could try to remove those anchors in other documents so that users would only use this method if they actually look specifically for it

Reply:
Under the hood, the `teleport-cluster` chart includes the `teleport-operator` chart. We use the same deployment method but add a nice "one value deployment" toggle for `teleport-cluster` users.